integration_tests/dbt_project.yml

```diff
@@ -21,5 +21,10 @@ vars:
   elementary_debug_logs: false
 
 models:
+  elementary_integration_tests:
+    +schema: "{{ env_var('CUSTOM_SCHEMA', 'default') | replace('-', '_') }}"
   elementary:
-    +schema: elementary
\ No newline at end of file
+    +schema: elementary
+
+seeds:
```

Comment:
Not entirely sure that the `seeds` should also use the custom schema, what do you think?  @oravi 
This will cause a lot of duplicated seeds but if seeds change (rarely), it won't override.